### ISSUE TITLE
[iOS] Metrics wifi check causes console spew when not connected to network

### ISSUE THREAD

**Issue by friedbunny** [mapbox/mapbox-gl-native#2378]
```
Sep 21 18:08:18 Hobbes wifid[75] <Error>: WiFi:[464566098.097497]: Network NULL for client configd !
Sep 21 18:08:18 Hobbes wifid[75] <Error>: WiFi:[464566098.971163]: Network NULL for client configd !
Sep 21 18:08:19 Hobbes wifid[75] <Error>: WiFi:[464566099.687127]: Network NULL for client configd !
Sep 21 18:08:20 Hobbes wifid[75] <Error>: WiFi:[464566100.536828]: Network NULL for client configd !
Sep 21 18:08:20 Hobbes wifid[75] <Error>: WiFi:[464566100.867812]: Network NULL for client configd !
Sep 21 18:08:21 Hobbes wifid[75] <Error>: WiFi:[464566101.329745]: Network NULL for client configd !
```

**Comment by friedbunny** [comment]
Oh, and by console I mean the device-wide console, not Xcode's.

**Comment by friedbunny** [comment]
@1ec5 suggested that we use Reachability here, which I'll investigate. Reachability doesn't offer the SSID, but may allow us to avoid the console spam if we wrap [`-wifiNetworkName`](https://github.com/mapbox/mapbox-gl-native/blob/feae692dbaefa51b893528a31139fb8282b3e889/platform/ios/MGLMapboxEvents.m#L740-L753) in a check.

**Comment by friedbunny** [comment]
Incidentally, [`CaptiveNetwork`](https://developer.apple.com/library/ios/documentation/SystemConfiguration/Reference/CaptiveNetworkRef/index.html) is deprecated in iOS 9 in favor of [`NEHotspotHelper`](https://developer.apple.com/library/ios/documentation/NetworkExtension/Reference/NEHotspotHelperClassRef/index.html), which requires special dispensation (an entitlement) from Apple in order to use. We would not qualify for this entitlement, as it's actually reserved for captive network setup.

**Comment by friedbunny** [comment]
Fixed by #2472.
